docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import SvgProductCore from 'docs/src/icons/SvgProductCore';
+import SvgProductAdvanced from 'docs/src/icons/SvgProductAdvanced';
+import SvgProductTemplates from 'docs/src/icons/SvgProductTemplates';
+import SvgProductDesign from 'docs/src/icons/SvgProductDesign';
+import SvgMuiX from 'docs/src/icons/SvgMuiX';
+
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+
+function ProductItem({
+  icon,
+  name,
+  description,
+}: {
+  icon: React.ReactNode;
+  name: React.ReactNode;
+  description: React.ReactNode;
+}) {
+  return (
+    <Box display="flex" alignItems="center" py={2}>
+      <Box px={2}>{icon}</Box>
+      <Box>
+        <Typography color="text.primary" variant="body2" fontWeight="bold">
+          {name}
+        </Typography>
+        <Typography color="text.secondary" variant="body2" fontWeight="regular">
+          {description}
+        </Typography>
+        <Typography
+          color="primary"
+          variant="body2"
+          fontWeight="bold"
+          sx={{ display: 'flex', alignItems: 'center', minHeight: 24 }}
+        >
+          <span>Learn more</span> <KeyboardArrowRightRounded fontSize="small" sx={{ mt: '2px' }} />
+        </Typography>
+      </Box>
+    </Box>
+  );
+}
+
+const ProductsSwitcher = () => {
+  const [productIndex, setProductIndex] = React.useState(0);
+  return (
+    <Tabs
```

Comment:
Can we have a hover state for the currently selected tab? In my demo, I used a blue border color and some box-shadow.